Classes/Settings/SettingsViewController.swift

```diff
@@ -259,3 +268,69 @@ NewIssueTableViewControllerDelegate {
         showDetailViewController(navigation, sender: nil)
     }
 }
+
+extension SettingsViewController {
+  // Default Reaction + MenuController
+  
+   private func showDefaultReactionMenu() {
+    
+    setDefaultReaction.becomeFirstResponder() // Required
+    
+    let actions = [
+      (ReactionContent.thumbsUp.emoji, #selector(SettingsViewController.onThumbsUp)),
+      (ReactionContent.thumbsDown.emoji, #selector(SettingsViewController.onThumbsDown)),
+      (ReactionContent.laugh.emoji, #selector(SettingsViewController.onLaugh)),
+      (ReactionContent.hooray.emoji, #selector(SettingsViewController.onHooray)),
+      (ReactionContent.confused.emoji, #selector(SettingsViewController.onConfused)),
+      (ReactionContent.heart.emoji, #selector(SettingsViewController.onHeart))
+    ]
+    
+    let menu = UIMenuController.shared
+    menu.menuItems = actions.map { UIMenuItem(title: $0.0, action: $0.1) }
+    menu.setTargetRect(defaultReactionLabel.frame, in: setDefaultReaction)
+    menu.setMenuVisible(true, animated: trueUnlessReduceMotionEnabled)
```

Comment:
Using `UIMenuController` to pick a selection in Settings feels really weird. Could we instead create a new `UITableViewController` that has all of the emoji options, and you select a cell?

Reply:
@rnystrom Context Menu type thing or you want a whole new view controller?

Reply:
While new view controller. Like how on settings sub menus are just a simple list:

![GitHawk Upload by rnystrom](https://i.imgur.com/YCAlwGr.jpg)